db/seeds.rb

```diff
@@ -1,3 +1,9 @@
+puts "Creating admin user"
```

Comment:
nice!

thoughts on putting together a quick smoke test for the seeds just to make sure they run OK? I feel like we've broken them a few times recently and not caught it until later on

Reply:
yeah, I was thinking about that, too! I think we should. I'll make a chore